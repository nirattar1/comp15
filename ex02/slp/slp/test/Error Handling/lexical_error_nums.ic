class Stam{
	int nums(){
		int a = -2147483647; // = -2^31 + 1
		int b = 2147483647; // = 2^31 - 1
		int c = 2147483648; // = 2^31
	}
}