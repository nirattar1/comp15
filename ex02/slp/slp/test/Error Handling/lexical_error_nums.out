5: Lexical error: integer out of bound '2147483648'